docs/development/Serial.md

```diff
@@ -93,11 +93,24 @@ Note: for Identifier see serialPortIdentifier_e in the source; for Function bitm
 | SERIAL_PORT_SOFTSERIAL2 | 31    |
 | SERIAL_PORT_LPUART1     | 40    |
 
-ID's 0-19 reserved for UARTS 1-20
-ID's 20-29 reserved for USB 1-10
-ID's 30-39 reserved for SoftSerial 1-10
-ID's 40-49 reserved for LPUART 1-10
-Other devices can be added starting from id 50.
+:::note
+
+- ID's 0-19 reserved for UARTS 1-20
+- ID's 20-29 reserved for USB 1-10
+- ID's 30-39 reserved for SoftSerial 1-10
+- ID's 40-49 reserved for LPUART 1-10
+- Other devices can be added starting from id 50.
+
+:::
+
+:::info
+
+In firmware 4.5 for SOFTSERIAL or LPUART we now for example have the following resources:
+
+resource SOFTSERIAL_TX 1 <PIN>
+resource LPUART_TX 1 <PIN>
```

Comment:
```suggestion
`resource LPUART_TX 1 <PIN>`
```

Reply:
resolves compile, but incorrect. i guess 3 backticks around the pair is more correct.
![image](https://github.com/betaflight/betaflight.com/assets/56646290/8f991d0a-0fd8-47c6-9d7e-b7e23ab1ff09)
--->
![image](https://github.com/betaflight/betaflight.com/assets/56646290/2d48e75f-dc75-405d-96c7-7ba009d23493)